guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -8,4 +8,30 @@
 == Upgrade Warnings
 
 [id="katello-deprecations"]
-== Deprecations
+== Removals
+
+=== Katello-Agent has been removed.
```

Comment:
I'd put this under Upgrade warnings.

And I'd also mention that it's automatically removed when upgrading, but that REX is not (automatically) installed as a replacement.

Reply:
committed!